### PR TITLE
Add github template to sharing page

### LINKED ISSUE #268: DOC: document GitHub template
### Describe the Issue

A new template was created, but it should be documented in the docs.

https://github.com/jeertmans/manim-slides-starter

### Affected Page

https://eertmans.be/manim-slides/reference/sharing.html

### Issue Type

Documentation Enhancement

### Recommended fix or suggestions

Add some words about the starter template.

### PR DESCRIPTION
**Pull request by taiyeoguns**
## Fixes Issue

Closes #268

## Description

Add information about Github template to sharing page

## Check List (Check all the applicable boxes)

- [x] I understand that my contributions needs to pass the checks.
- [ ] If I created new functions / methods, I documented them and add type hints.
- [ ] If I modified already existing code, I updated the documentation accordingly.
- [x] The title of my pull request is a short description of the requested changes.

### CODE REVIEW

File: docs/source/reference/sharing.md [hunk 1]
```diff
@@ -148,6 +148,11 @@ to know how to share your slide with GitHub pages, see the
 can take some time, and *glitches* may occur between slide transitions for this
 reason.
 
+
+### Using the Github starter template
+
+A [starter template](https://github.com/jeertmans/manim-slides-starter) is available which allows to quickly get going with a new Manim slides presentation on your Github account. The template comes ready with functionality to automate tasks using Github actions and publish to Github Pages. Please refer to the template page for usage instructions.
+
```
Comment (jeertmans) [inline docs/source/reference/sharing.md:155]: Can you split lines such that they never make more than 81 characters wide?

This is more a style requirements, but makes reading the code easier :-)

**Review by jeertmans — CHANGES_REQUESTED** [review]

**Comment by jeertmans** [comment]
Hello @taiyeoguns, thanks for your PR! I just left one comment :-)

**Comment by taiyeoguns** [comment]
Thanks @jeertmans, made the update

**Comment by jeertmans** [comment]
Thanks a lot!

**Pull request merged**